tests/Microsoft.Data.DataFrame.Tests/DataFrame.IOTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.Data.Tests
+{
+    public partial class DataFrameTests
+    {
+        [Fact]
+        public void TestReadCsv()
```

Comment:
We probably need way more Read Csv tests than this.... but this is fine for now. Can you log an issue to add more tests in this area?

Reply:
Yup, I agree. Filed https://github.com/dotnet/corefxlab/issues/2713